arches_lingo/apps.py

```diff
@@ -0,0 +1,7 @@
+from django.apps import AppConfig
+
+
+class Arches_LingoConfig(AppConfig):
+    name = "arches_lingo"
+    verbose_name = "Arches_Lingo"
```

Comment:
This should be a human-readable name. For that reason, we've removed it from the template, but may as well pick one here and use it. Shows up in the django admin, I think.

```suggestion
    verbose_name = "Arches Lingo"
```